pkg/platform/aws/aws.go

```diff
@@ -109,6 +109,7 @@ func NewConfig() *config {
 		Region:             "eu-central-1",
 		EnableAggregation:  true,
 		EnableTLSBootstrap: true,
+		NetworkMTU:         1500, //nolint:gomnd
```

Comment:
It seems this could be a constant. Perhaps we could put it in `platform` package?